REFERENCE.md

```diff
@@ -382,103 +383,103 @@ Port the service should listen on. Module default is `undef` which means that po
 isn't added to chrony.conf, and chrony listens to the default ntp port 123 if
 `queryhosts` is used.
 
-Default value: `undef`
+Default value: ``undef``
```

Comment:
some values have a single backtick, some have two? never saw that in the past in a REFERENCE.md

Reply:
Odd.  Seems to render ok though.

Reply:
I guess related to https://github.com/puppetlabs/puppet-strings/pull/243

Reply:
> prevent emoji from being inadvertantly rendered
lol